jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBQueryResultSet.java

```diff
@@ -726,7 +730,7 @@ public boolean next() throws SQLException {
     if (maxRowsOrRowsLimit > 0 && rowsFetched >= maxRowsOrRowsLimit) {
       // The constraint of maxRows instead of rowsLimit is embodied
       if (rowsLimit == 0 || (maxRows > 0 && maxRows < rowsLimit)) {
-        System.out.println("Reach max rows " + maxRows);
+        LOGGER.info("Reach max rows " + maxRows);
```

Comment:
"info" may be too detailed here, I would suggest using "debug".

Reply:
ok